src/importlinter/contracts/forbidden.py

```diff
@@ -53,6 +52,13 @@ def check(self, graph: ImportGraph) -> ContractCheck:
                     importer=source_module.name, imported=forbidden_module.name
                 )
                 if chains:
+                    if self.allow_indirect_imports and str(self.allow_indirect_imports) in (
+                        "True",
+                        "true",
+                    ):
```

Comment:
NIP: or just: `str(self.allow_indirect_imports).lower() == 'true'` when we want to support only `True` as a truthy value

Reply:
Sure was just following the other method in this class that evaluated a boolean string.